setup.py

```diff
@@ -12,7 +12,6 @@
       install_requires=[
```

Comment:
Should there be an additional `extra_requires` for the `pyinstaller` stuff?

Reply:
I don't think so. The pyinstaller is for packaging up terra, but never for "running" it. I don't usually see a `dev` extras, if that's what you are thinking? Same for autopep8, coverage, etc... Typically I see packages have a `requirements.txt` which mirrors `setup.py` and `requirements_dev.txt` for building the wheel/etc. The Pipfile replaces both of those. Plus, eventually I want to use pyinstalller via a docker, not by installing it in the host virtualenv, if I ever get that working.